bitmapist/cohort/tmpl/table_data.mako

```diff
@@ -97,7 +97,8 @@
                 for row_data in dates_data:
                     prct = row_data[i]
                     cnts += 1
-                    total += prct
+                    if prct:
+                        total += prct
```

Comment:
I think probably something like this would be more correct/defensive:

```python
if prct != '':
    cnts += 1
    total += prct
```

Reply:
That's exactly what we had before https://github.com/Doist/bitmapist/commit/c577e303b720dc638a2fcdfccb775184e951e476. We still need to increment `cnts`

Reply:
@imankulov I don't think it's the same, because this won't ignore the `0` case 👀 , while https://github.com/Doist/bitmapist/commit/c577e303b720dc638a2fcdfccb775184e951e476 will

Reply:
It seems both implementations, the one Roman is submitting and what Amir said on the comment above, will have the same result. `prct != ''` will consider `prct = 0`, but it will generate a zero-sum, what doesn't change the result. In both scenarios `cnts` is being incremented anyway.

Other than that, I would even suggest another improvement. Does `cnts` means `counts`? If so, why not make `counts = len(dates_data)`?